integrations/serverpod_cloud_storage_gcp/lib/src/aws_s3_upload/aws_s3_upload.dart

```diff
@@ -61,9 +61,13 @@ class GCPS3Uploader {
     try {
       final res = await req.send();
 
+      if (res.statusCode >= 400 && res.statusCode < 500) {
+        print('Failed to upload to GCP, with reason: ${res.reasonPhrase}');
```

Comment:
Question: Shouldn't these be `stdout`/`stderr`?

Reply:
Maybe a good idea? I followed the existing logging in the file that is using print. Should I change them all?

Reply:
Might as well, unfortunate that we don't have a proper server logging system.